### ISSUE TITLE
readASD

### ISSUE THREAD

**Issue by l-ramirez-lopez** [l-ramirez-lopez/prospectr#2]
further update the readASD function for full compatibility with v8 files (re-organize it)

**Comment by pierreroudier** [comment]
Hey @l-ramirez-lopez ,

Just bringing your attention to [our `asdreader` package](https://github.com/pierreroudier/asdreader) -- which sole and only purpose is to precisely read ASD files.

Happy to get you on-board if you want to contribute to the package, and use it within `prospectr`, as I'm concerned of duplicated efforts. 

Let me know what you think!

**Comment by l-ramirez-lopez** [comment]
Hi Pierre,
Sounds a really good idea. I'll be happy to contribute.
I see your package appears to have one minor issue reported (which might be not even an actual bug). So please let me know how you think I can contribute.

**Comment by pierreroudier** [comment]
> I see your package appears to have one minor issue reported (which might be not even an actual bug). 
> 

Yes, we have [this issue](https://github.com/pierreroudier/asdreader/issues/6) open for about 2 years, because I can't reproduce it. It sounds like a limitation of Windows (hard limit on the number of files open?).

> So please let me know how you think I can contribute.
> 

The `asdreader` package works well as is, so I'm sure you know, most of the work is un-glamourous maintenance! I think something very valuable would be to:

1. Ensure we have the same features as [your readASD function](https://github.com/l-ramirez-lopez/prospectr/blob/master/R/readASD.R).
2. Ensure the different versions of ASD files are supported -- a very useful addition would be some unit testing specifically on this. 

Thanks!
